### DISCUSSION TITLE
Gluco Check Spanish (DONE) ✅

### DISCUSSION THREAD

**Discussion by nielsmaerten** [nielsmaerten/gluco-check#142] [General]
## Update 2022-01-06
The Spanish version of Gluco Check is now available 🎉
Thanks to everyone who helped make this possible!

🥇 @JavierDT1 : Translating + Testing
🥇 @mgoldens : Testing + Translation review
🥇 @friquelme98 : Translation review
🥇 @IterCriminis : Testing

---

Hi! We're busy rolling out support for Spanish in Gluco Check. All strings have been translated, but we still need someone to: 
* [x] [proofread the translation](https://translate.glucocheck.app)
* [x] test the Spanish version 
* [x] report any bugs

Do you speak both Spanish and English? Do you want to help?
Please leave a comment below to get in touch.

Thank you! 👏
niels
david

PS: If you prefer email, you can reach us at: translations@glucocheck.app

**Comment by JavierDT1** [comment]
Hi. I speak both Spanish and English. I can help you.

**Comment by IterCriminis** [comment]
I speak both too, if you need help on translate i can help

**Reply by nielsmaerten** [reply]
Thanks for reaching out! If you send an email to translations@glucocheck.app, I'll add you to our list of testers.

The Spanish translation is currently undergoing a (hopefully) final review by Google. If things go well, it may be generally available by the end of this week 🤞. I'll update this post when I get feedback from Google.

(Just don't hold your breath... these reviews are notoriously complex ;)
